library/core/src/primitive_docs.rs

```diff
@@ -895,24 +916,11 @@ mod prim_str {}
 ///
 /// # Trait implementations
 ///
-/// If every type inside a tuple implements one of the following traits, then a
-/// tuple itself also implements it.
-///
-/// * [`Clone`]
-/// * [`Copy`]
-/// * [`PartialEq`]
-/// * [`Eq`]
-/// * [`PartialOrd`]
-/// * [`Ord`]
-/// * [`Debug`]
-/// * [`Default`]
-/// * [`Hash`]
```

Comment:
I like the explicitness of the old verbiage here. It may not strictly be needed with the new, improved display of trait implementations, but it's an alternate way to present the material and I think it's worth keeping.